01-basics/04-MeetupTitle/script.js

```diff
@@ -14,4 +14,71 @@ function fetchMeetupById(meetupId) {
   });
 }
 
-// Требуется создать Vue приложение
+const template = `
+  <div>
+    <label v-for="n of 5" :key="n">
+      <input
+        type="radio"
+        :value="n"
+        v-model="meetupId"
+        :disabled="loading"
+      /> {{ n }}
+    </label>
+    <hr />
+
+    <h3 v-show="!loading">{{ meetupHeader }}</h3>
+    <h4 v-show="loading">loading...</h4>
```

Comment:
`v-show` hides nodes from user with `display: none` but the node is still rendered. It's better to use `v-if` here.